dlt/destinations/impl/databricks/__init__.py

```diff
@@ -0,0 +1,60 @@
+from typing import Type
+
+from dlt.common.schema.schema import Schema
+from dlt.common.configuration import with_config, known_sections
+from dlt.common.configuration.accessors import config
+from dlt.common.destination import DestinationCapabilitiesContext
+from dlt.common.destination.reference import JobClientBase, DestinationClientConfiguration
+from dlt.common.data_writers.escape import escape_databricks_identifier
+from dlt.common.arithmetics import DEFAULT_NUMERIC_PRECISION, DEFAULT_NUMERIC_SCALE
+
+from dlt.destinations.impl.databricks.configuration import DatabricksClientConfiguration
+
+
+@with_config(
+    spec=DatabricksClientConfiguration,
+    sections=(
+        known_sections.DESTINATION,
+        "databricks",
+    ),
+)
+def _configure(
+    config: DatabricksClientConfiguration = config.value,
+) -> DatabricksClientConfiguration:
+    return config
+
+
+def capabilities() -> DestinationCapabilitiesContext:
+    caps = DestinationCapabilitiesContext()
+    caps.preferred_loader_file_format = "parquet"
+    caps.supported_loader_file_formats = ["parquet"]
```

Comment:
I could not get `jsonl` to work with `COPY INTO`. Suspect it's because of gzip, i.e. either databricks doesn't support reading gzipped json or more likely requires file extension `json.gz`

Reply:
can you confirm if using a different extension works?

Reply:
https://docs.databricks.com/en/sql/language-manual/delta-copy-into.html

Reply:
The multiline mode applies to records that span multiple lines, jsonl works otherwise by default but only with gz compression disabled.    
I have to try some script with `jsonl.gz` filename, looks like a lot of work to actually change extensions in dlt.

Reply:
Confirmed that gzip works with `jsonl.gz` extension.   
Might be good to change this so dlt creates files with `gz` ext, that's pretty universally supported, but would leave it for another pr.  
I just have parquet enabled for now.

Reply:
@steinitzu my 2 cents:
there's really no way to set the compression explictly ie. in copy or format options?
if not then please do not disable JSONL, just detect that compression is enabled and fail the job with an exception (ie. ask to disable compression)

Reply:
another ask @steinitzu could you write a ticket to add `gz` extensions to zipped files in dlt load packages?

Reply:
> @steinitzu my 2 cents: there's really no way to set the compression explictly ie. in copy or format options?

Hey @rudolfix no there's no option for it that's documented.

>  if not then please do not disable JSONL, just detect that compression is enabled and fail the job with an exception (ie. ask to disable compression)

Hmm yeah that makes sense for now. Until we change the extensions.

Reply:
It fails on empty jsonl files too. Produced by this test: https://github.com/dlt-hub/dlt/blob/sthor%2Fdatabricks/tests/load/pipeline/test_replace_disposition.py#L154-L163  
Is this supposed to produce a stage file? Not sure if I'm doing something wrong.

Reply:
yes, empty jsonl files are correct. they mean that content of the table should be dropped as there are no rows in current load. we generate them ie. to delete data from child tables on replace if there were no data at all for given table. your workaround is good and will do what we expected! thanks